File: SW/D3_1217.py
#재귀호출 사용
def check(a, b, c):
    if b == c:
        return a
    
    return a * check(a, b, c+1)

for _ in range(10):
    t = int(input())
    n, m = map(int, input().split())
    res = check(n, m, 1)
    print("#{} {}".format(t, res))

File: SW/D3_1217.java
import java.util.Scanner;

public class D3_1217 {
    static int res;
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        for (int t = 1; t <= 10; t++) {
            int tmp = sc.nextInt();
            int N = sc.nextInt();
            int M = sc.nextInt();

            res = N;
            check(N, M-1);
            System.out.printf("#%d %d\n", t, res);
        }
    }

    static void check(int n, int d) {
        if (d == 0) return;
        res *= n;
        check(n, d-1);
    }
}
